File: generate_better_results/generate_better_shots.m
close all; clear;

aux= load('recdata.mat').rec;
aux= aux(1:25:end,1:2:end); %reduzi para uma dimensão legal (837,674);


shot2 = make_better_shots(aux);

function shot2 = make_better_shots(shot)
    limiar = 1e-7;
    positions = shot <= limiar; %this variable is logical
    shot2 = shot;
    
    shot2(positions) = shot2(positions) +0.6;
    
    
    figure(1);imshow(shot);
    figure(2);imshow(shot2);
    
end

File: olds/generate_better_results/generate_better_shots.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Fri Mar  6 15:15:01 2020

@author: jonas
"""

import scipy.io as sio
import matplotlib.pyplot as plt


shot = sio.loadmat('recdata.mat');
shot = shot['rec']; #dimensão (20904,1348)
shot = shot[1:-1:25,1:-1:2] # dimensão (837,673)



def generate_better_results(shot):
        
    limiar = 1e-7
    
    
    positions = shot <= limiar; #this variable is logical
    
    shot[positions] = shot[positions] +0.6;
    
    
    plt.imshow(shot, cmap='binary')



generate_better_results(shot)